roommanager.js

```diff
@@ -132,11 +132,20 @@ module.exports = function (server, storage) {
 			length: 0
 		};
 
-		queueItem.service = InfoExtract.getService(link);
+		if (typeof video === "string") {
```

Comment:
Following the above comment, inconsistency in the data leads to these hacky solutions. This should check if a url is provided in the object and redirecting to other handlers if it is not.

Reply:
That's a good point, I agree. I'll rework it